tests/geobase/tests_FairGeoShapes.cxx

```diff
@@ -0,0 +1,408 @@
+/********************************************************************************
+ *   Copyright (C) 2024 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH     *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+
+#include "FairGeoAssembly.h"
+#include "FairGeoBasicShape.h"
+#include "FairGeoBrik.h"
+#include "FairGeoCone.h"
+#include "FairGeoCons.h"
+#include "FairGeoEltu.h"
+#include "FairGeoPcon.h"
+#include "FairGeoPgon.h"
+#include "FairGeoSphe.h"
+#include "FairGeoTorus.h"
+#include "FairGeoTrap.h"
+#include "FairGeoTrd1.h"
+#include "FairGeoTube.h"
+#include "FairGeoTubs.h"
+#include "FairGeoVolume.h"
+#include "checks_FairGeoBasicShape.h"
+#include "generate_TestInput.h"
+
+#include <string>
+#include <vector>
+
+#ifndef TESTS_GEOBASE_FAIRGEOBASICSHAPE_H
+#define TESTS_GEOBASE_FAIRGEOBASICSHAPE_H
```

Comment:
Why those header guards in a `.cxx`?

While you're here, you might simplify some code below by using this?

```suggestion

using fairroot::tests::check_FairGeoShape_Construction;
```